app/workers/go_project_verification_worker.rb

```diff
@@ -21,6 +21,17 @@ def perform(name)
       # if we can't get a canonical name then bail out of here and we'll have to investigate further
       return if canonical_name.blank?
 
+      unless name.downcase.include?(canonical_name.downcase)
```

Comment:
On line 38 below, it looks like we had a pre-existing stricter check that it actually has to be equal (except for case) and not a substring. :thinking: 

After this PR we'd have this comparison in 3 places, and it's different in one (the existing one on line 38 below). That probably means it should be somehow factored out.

Reply:
Ah didn't notice that. I added this purely to log the name diffs if they didn't match wherever we were using the `canonical_module_name` to get some data on the size of the problem. Since the check below is stricter and also logs, it might be better to just remove this logging entirely and look at the existing data we have.